teams/vsp/product-management/product-brief-template.md

```diff
@@ -1,31 +1,36 @@
 # **VSP Product Brief Template**
 
-### What is it?
+To create a Product Brief:
+1. Copy this file to `platform/<product-name>/README.md`.
+1. Fill out all sections below for your product.
+1. Delete this preamble, all italicized instruction blocks, and any unused links within the template.
 
-*Explain how this product will benefit VFS teams.*
+# Product Title
 
-### How does it work?
-
-*Describe at a high level how the product will impact VFS teams' workflow.*
+*Add 1-2 sentences about what this product is and how it will benefit VFS teams.*
 
 ------
 
-### Product documentation
+## Documentation
+
+*Add links to supporting documentation and details about what value those docs bring to VFS teams.*
 
-*Add links to supporting documentation.*
+* User Guide
+* Actual tool _(if relevant)_
+* Status dashboard _(if relevant)_
 
```

Comment:
```suggestion
* Product Design Document
* User Guide
* Actual tool _(if relevant)_
* Status dashboard _(if relevant)_

```

Reply:
-1 for two reasons:
- design docs _might_ connect to Product, but generally they're more connected to Initiatives. 
- this README is VFS-facing for "what is this and how to use it", design docs are more VSP-facing for "how does this work / why did we implement it this way"